docs/preview/features/endpoint-validation.md

```diff
@@ -5,8 +5,10 @@ layout: default
 
 # Endpoint validation
 
-We provide support for endpoint validation, when implementing your own custom web hook. This validation allows to secure your web hook with a secret key (taken from the query string or an HTTP header).  
-This is needed, because Azure Event Grid is send a validation request to a newly configured web hook, in order to prevent people leveraging Azure Event Grid to bring down a 3rd party API. 
+We provide support for endpoint validation, when implementing your own custom web hook.
```

Comment:
In all honesty, I'm not sure if I get the difference - One is the validation handshake for a new subscription, the other for authorization based on a key?

If so, why do we have a special attribute for the key and not use Arcus WebAPI for this?

Reply:
To be honest, I did an as-is move towards this new project and use the secret store. That was my remark here #164 . Further than that, I didn't want to move to far ahead. I guess we should ask @SamVanhoutte why this was split into two attributes?